sessions/workshops/natural-language-processing-in-10-lines-of-code.md

```diff
@@ -1,6 +1,6 @@
 speaker: andraz-hribernik
 title: Natural Language Processing in 10 Lines of Code
 ---
-Do you know that there are over 150 personal names mentioned in Jane Austen's Pride & Prejudice? Did you ever want to remove common words such as 'we', 'what', and 'for' from text? Or extract interesting keywords from it? Did you know all of this could be done in 10 lines of Python code?
-
-In this workshop we will answer these and similar questions using the open source library spaCy. One hour spent with us will be useful for anybody who is interested in text processing, text information extraction or text related data science. You do not need to have prior experiences with NLP or data science. However, some Python programming skills are needed. We would also encourage you to install spaCy in advance (link to Github repo with installation instructions).
\ No newline at end of file
+Do you know that there are over 150 personal names mentioned in Jane Austen's Pride & Prejudice? Did you ever want to remove common words such as 'we', 'what', and 'for' from text? Or how to extract interesting keywords from a document? Did you know all of this could be done in 10 lines of Python code?
+In this workshop we will answer these and similar questions using the open source library [spaCy](http://spacy.io/). One hour spent with us will be useful for anybody who is interested in text processing, text information extraction or text related data science. You will leave with the tools and knowledge to start developing your own natural language processing projects in Python.
```

Comment:
It looks like you’re trying to start a new paragraph – if so, you’ll need to add an extra blank line between the paragraphs.